rotkehlchen/accounting/structures/base.py

```diff
@@ -197,7 +197,7 @@ def entry_type(self) -> HistoryBaseEntryType:
 
     def get_type_identifier(self, **kwargs: Any) -> str:  # pylint: disable=unused-argument
         """Get the type identifier for this event. Subclasses may accept additional arguments"""
-        return str(self.event_type) + '__' + str(self.event_subtype)
+        return str(self.event_type) + '_' + str(self.event_subtype)
```

Comment:
here you should use the function `get_accounting_rule_identifier` you just made above.